pkg/agent/openflow/pipeline.go

```diff
@@ -1853,6 +1853,25 @@ func (c *client) localProbeFlow(localGatewayIPs []net.IP, category cookie.Catego
 	return flows
 }
 
+// snatSkipNodesFlows installs the default flows for skip SNAT for traffic to
+// the k8s nodes.
```

Comment:
skipping

K8s

Nodes

Reply:
the default flows for skip -> a flow to skip

Reply:
I will fix this.